tools/monitoring/mysql/db_mysql.go

```diff
@@ -71,3 +75,42 @@ func (c DBConfig) dataStoreName(dbName string) string {
 
 	return fmt.Sprintf("%sunix(%s)/%s", cred, "/cloudsql/"+c.Instance, dbName)
 }
+
+// PubsubMsgHandler adds record(s) to ErrorLogs table in database,
+// after parsing build log and compares the result with config yaml
+func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumber int) error {
+	config, err := config.ParseYaml(configURL)
+	if err != nil {
+		return err
+	}
+
+	logs, err := logs.ParseLog(buildLogURL, config.CollectErrorPatterns())
+	if err != nil {
+		return err
+	}
+
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
+	stmt, err := tx.Prepare("INSERT INTO ErrorLogs VALUES()")
+
+	for _, errorLog := range logs {
+		_, err := stmt.Exec(errorLog.Pattern, errorLog.Msg, jobname, prNumber, buildLogURL, time.Now())
+		if err != nil {
+			return err
+		}
+	}
+
+	err = stmt.Close()
```

Comment:
combine the if condition

if err = stmt.Close(); err != nil {
return err
}

Reply:
This line is too long to be combined.

Reply:
Could you elaborate what you mean by the line is too long to be combined? Did you get a golint error or something similar?

Reply:
should this be `defer stmt.Close()`